recipes/rpy2/build.sh

```diff
@@ -10,7 +10,6 @@ if [ "$(uname)" == "Darwin" ]; then
     export CFLAGS="${CFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
     export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
     export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
-    export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
```

Comment:
Yep. Only solution I can think of that would work everywhere is to use something like [spack's compiler wrappers](http://spack.readthedocs.io/en/latest/packaging_guide.html?highlight=compiler#compiler-wrappers)